lib/components/AlbumScreen/song_menu.dart

```diff
@@ -170,6 +180,62 @@ class _SongMenuState extends State<SongMenu> {
     }
   }
 
+  Future<bool> shouldShowSpeedWidget(currentSpeed) async {
+    if (currentSpeed != 1.0 ||
+        FinampSettingsHelper.finampSettings.contentPlaybackSpeedType.index ==
+            1) {
+      return true;
+    }
+    if (FinampSettingsHelper.finampSettings.contentPlaybackSpeedType.index ==
+        0) {
+      var genres = widget.item.genres!;
+
+      for (var i = 0; i < genres.length; i++) {
+        if (["audiobook", "speech"].contains(genres[i].toLowerCase())) {
+          return true;
+        }
+      }
+
+      try {
+        var parent =
+            await _jellyfinApiHelper.getItemById(widget.item.parentId!);
+        if (parent.runTimeTicks! > 72e9.toInt()) {
+          return true;
+        }
+      } catch (e) {
+        errorSnackbar(e, context);
+      }
+    }
+
+    return false;
+  }
+
+  final inputStep = 0.9;
+  var oldExtent = 0.0;
```

Comment:
Use the value from line 250 (`initialChildSize`) here. You'll have to define it as a variable.

Reply:
Yes, I did it for you :)